docs/Roadmap.md

```diff
@@ -0,0 +1,150 @@
+
+---
+
+### Expanded platform compatibility
+ONNX Runtime already supports a wide range of architectures, platforms, and languages, and this will continue to be an active investment area to broaden the availability of the engine for varied usage. 
+Additionally, we understand that lightweight devices and local applications may have constraints for package size, so there is active awareness to opportunistically minimize binary size.
+
+#### Architectures
+|Supported|Future|
+|---|---|
+|X64|Additional as-needed|
+|X86| |
+|ARM64 (Limited)| |
+|ARM32 (Limited)| |
+
+#### Platforms
+|Supported|Future|
+|---|---|
+|Windows 7+|Android (community contribution, in progress)|
+|Linux (various)|iOS|
+|Mac OS X| |
+
+#### Languages
+|Supported|Future|
+|---|---|
+|Python (3.5, 3.6, 3.7)|Java (community contribution, in progress)|
+|C++| |
+|C#| |
+|C| |
+|Ruby (community project)| |
+
+
+### Accelerators and Execution Providers
+#### New EPs
+To achieve the best performance on a growing set of compute targets across cloud and the intelligent edge, we invest in and partner with hardware partners and community members to add new execution providers. The flexible pluggability of ONNX Runtime is critical to support a broad range of scenarios and compute options.
+
+|Supported|Future|
+|---|---|
+|MLAS (Microsoft Linear Algebra Subprograms)|Android NN API (in progress)|
+|Intel MKL-DNN / MKL-ML|Qualcomm (TBD)|
+|Intel nGraph|AMD (TBD)|
+|NVIDIA CUDA| |
+|NVIDIA TensorRT| |
+|Intel OpenVINO| |
+|Nuphar Model Compiler| |
+|Microsoft Direct ML| |
+
+#### CUDA operator coverage
+To maximize performance potential, we will be continually adding additional CUDA implementations for supported operators.
+
+#### Simplify EP contributions
+In addition to new execution providers, we aim to make it easy for community partners to contribute in a non-disruptive way. To support this, we are investing in improvements to the execution provider interface for easily registering new execution providers and separating out EPs from the core runtime engine.
+
+### Continued Performance Optimizations
+Performance is a key focus for ONNX Runtime. From latency to memory utilization to CPU usage, we are constantly seeking strategies to deliver the best performance. Although DNNs are rapidly driving research areas for innovation, we acknowledge that in practice, many companies and developers are still using traditional ML frameworks for reasons ranging from expertise to privacy to legality. As such, ONNX Runtime is focused on improvements and support for both DNNs and traditional ML. 
+
+#### Examples of projects the team is working on:
+* Improvements to batch processing for scikit-learn models
+* More quantization support
+* Multithreading
+* Graph optimizations
+* Intelligent graph partitioning to maximize the value of different accelerators
+* Optimizations for mobile and IoT Edge devices
```

Comment:
### Optimizations for IoT:
IoT is a growing opportunity to execute ML workloads on the edge of the network, where data is collected. But the devices used for ML execution have different HW specifications. We will invest in areas for optimizing ONNX model execution across the breadth of IoT endpoints using different HW configurations with CPUs, GPUs and custom NN ASICs.